scripts/check_imports.swift

```diff
@@ -99,7 +99,9 @@ func checkFile(_ file: String, isPublic: Bool) {
       let importFile = line.components(separatedBy: " ")[1]
       if inSwiftPackageElse {
         if importFile.first != "<" {
-          genError("Import error: \(file):\(lineNum) Import in SWIFT_PACKAGE #else should start with \"<\".")
```

Comment:
Looks like your local swiftformat might be the wrong version - see https://github.com/firebase/firebase-ios-sdk#code-formatting or just revert this change.